infrastructure/terraform.test.tfvars

```diff
@@ -20,17 +20,13 @@ public_subnet_cidr_block = {
   "eu-west-1a" = "15.0.1.0/24"
   "eu-west-1c" = "15.0.2.0/24"
 }
-<<<<<<< HEAD
```

Comment:
@sdebruijn can you please take a pull in your branch as I see this issue is not exist anyone

Reply:
I'm not quite used to this process. Do I need to pull from XgridInc/xc3:main? I think my editor shows it is up-to-date with that branch. Am I doing something wrong?

Reply:
I think it is resolved